### ISSUE TITLE
Crash in EziSocialManager.mm

### ISSUE THREAD

**Issue by j-jorge** [ezibyte/EziSocial-Plugin#9]
Hi,

The following crash happens randomly in the EziSocial plugin when I am connected to Facebook in my iphone app. Could you check the line 2609 of `EziSocialManager.mm` to see what would fail?

```
objc_msgSend() selector name: objectForKey:

0   libobjc.A.dylib                      0x3b64b626 objc_msgSend + 6
1   Bazoo Block                          0x0013ec9d -[EziSocialManager updateUserInfo] (EziSocialManager.mm:2609)
2   Foundation                           0x3188c117 __NSFireDelayedPerform + 412
3   CoreFoundation                       0x30e77f47 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 12
4   CoreFoundation                       0x30e77b63 __CFRunLoopDoTimer + 792
5   CoreFoundation                       0x30e75eeb __CFRunLoopRun + 1216
6   CoreFoundation                       0x30de07a9 CFRunLoopRunSpecific + 522
7   CoreFoundation                       0x30de058b CFRunLoopRunInMode + 104
8   GraphicsServices                     0x35d4d6d3 GSEventRunModal + 136
9   UIKit                                0x3373f891 UIApplicationMain + 1134
```

**Comment by ezibyte** [comment]
Checking...

**Comment by ezibyte** [comment]
It looks like parsing it getting failed because of data unavailability. We will check it and release the patch by tomorrow.

**Comment by j-jorge** [comment]
Hi,

Any news about the fix for this bug?

**Comment by ezibyte** [comment]
Yes, it is fixed but will be pushed in upcoming release. EziSocial 2.3 (this Sunday). 20 Apr

**Comment by ezibyte** [comment]
Fixed in EziSocial 2.3 release.
